### PR TITLE
Support TVM2 in cocaine-tool

### PR DESCRIPTION
**Pull request by kletkavrubashku**

### CODE REVIEW

File: cocaine/tools/actions/keyring.py [hunk 1]
```diff
@@ -27,6 +31,9 @@ def execute(self):
 
 class Update(Action):
     def __init__(self, cid, tvm):
+        if tvm.ty() != TVM.ty():
```
Comment (shaitan) [inline cocaine/tools/actions/keyring.py]: Why isn't it checked via `isinstance(tvm, TVM)`?

File: cocaine/tools/actions/keyring.py [hunk 2]
```diff
@@ -79,6 +86,9 @@ def execute(self):
 
 class Refresh(Action):
     def __init__(self, tvm):
+        if tvm.ty() != TVM.ty():
+            raise Exception("action is available only for mode: '{}'".format(TVM.ty()))
```
Comment (shaitan) [inline cocaine/tools/actions/keyring.py]:
```python
"{} doesn't support the refresh action".format(tvm.ty())
```

File: cocaine/tools/actions/keyring.py [hunk 3]
```diff
@@ -27,6 +31,9 @@ def execute(self):
 
 class Update(Action):
     def __init__(self, cid, tvm):
+        if tvm.ty() != TVM.ty():
+            raise Exception("action is available only for mode: '{}'".format(TVM.ty()))
```
Comment (shaitan) [inline cocaine/tools/actions/keyring.py]:
```python
"{} doesn't support the update action".format(tvm.ty())
```

File: cocaine/tools/actions/keyring.py (hunk 1 above)
Comment (kletkavrubashku) [reply cocaine/tools/actions/keyring.py]: Reverted, because actions with keyring use "tvm" service directly.

**Review by shaitan — APPROVED** [review]

**Pull request merged**